subprojects/docs/src/docs/dsl/org.gradle.jvm.application.tasks.CreateStartScripts.xml

```diff
@@ -10,6 +10,7 @@
             <tr><td>classpath</td></tr>
             <tr><td>defaultJvmOpts</td></tr>
             <tr><td>outputDir</td></tr>
+            <tr><td>scriptDir</td></tr>
```

Comment:
Why this is `scriptDir` instead of `executableDir`?

Reply:
I was trying to make it consistent with CreateStartScripts.java which has this line:
```
generator.setScriptRelPath(getScriptDir() + "/" + getUnixScript().getName());
```